app/pages/resource/resource-auth-mapping.js

```diff
@@ -0,0 +1,30 @@
+import facebookIcon from '../../assets/icons/facebook.svg';
+import googleIcon from '../../assets/icons/google.svg';
+import mobileIcon from '../../assets/icons/mobile.svg';
+import suomiFiIcon from '../../assets/icons/suomiFi.svg';
+import yleIcon from '../../assets/icons/yle.svg';
+
+/*
+Mapping of resource authentication level with the corresponding allowed login methods.
+If this mapping is updated backend mapping needs updating too and vice version.
+The order of loginMethodNames and loginMethodIcons should match since the title/alt for
+icons is used from loginMethodNames.
+*/
+export const RESOURCE_AUTHENTICATION_GROUPING = {
```

Comment:
Can you add a second comment with a short explanation on the different levels and how it works for other developers sake in the future.

Reply:
Added.